content/en/blog/2024/sig-api-machinery-spotlight.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: blog
+title: "Spotlight on SIG API Machinery"
+slug: sig-api-machinery-spotlight-2024
+date: 2024-06-04
+author: "Frederico Muñoz (SAS Institute)"
+---
+
+We recently talked with [Federico Bongiovanni](https://github.com/fedebongio) (Google) and [David
+Eads](https://github.com/deads2k) (Red Hat), Chairs of SIG API Machinery, to know a bit more about
+this Kubernetes Special Interest Group.
+
+## Introductions
+
+**Frederico (FSM): Hello, and thank your time. To start with, could you tell us about
```

Comment:
little typo: thank you for your time

Reply:
Apart from this minor fix, it looks good to me.

Reply:
ty, fixed in last commit.